doc/source/images/generate_diagram.py

```diff
@@ -28,7 +28,6 @@ def get_rect_height(name, obj):
     nlines = 1.5
     nlines += len(getattr(obj, '_all_attrs', []))
     nlines += len(getattr(obj, '_single_child_objects', []))
-    nlines += len(getattr(obj, '_multi_child_objects', []))
```

Comment:
This script is currently not running due to some imports not being present. I think it would help to also include the generated diagram to understand the structural changes here.

Reply:
@apdavison  : do you plan to tedisouly also fix the diagram generator or will we fix this later ?